src/examples/css/style.css

```
@@ -1,10 +1,10 @@
 /* Default Styling */
 
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
```

Stylelint (17.10.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"" (import-notation)

(import-notation)